### PR TITLE
RF [TC_06_24_01]

### PR DESCRIPTION
**Pull request by IrenaKh95**
- тест был перенесен в documentsPage (т.к. изначально был написан на неверной стр. - templatesActivePage);
- была удалена страница  page_objects/templatesActivePage.js;
- в fixtures - добавлено возвращение на SignPage после создания папки;
- после выполнения теста было добавлено возвращение на SignPage.

### CODE REVIEW

File: tests/06Folders.spec.js [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
 
-        await expect(templatesActivePage.locators.getToaster()).toHaveText('Folder deleted successfully.');
+        await documentsPage.clickOptionsBtn();
+        await documentsPage.clickDeleteBtn();
+        await documentsPage.clickYesDeleteBtn();
+        await documentsPage.locators.getToaster().waitFor({ state: 'visible' });
+
+        await expect(documentsPage.locators.getToaster()).toHaveText('Folder deleted successfully.');
+
+        await documentsPage.clickSignaturelyLogoAndGoSignPage();
```
Comment (S-Lanchik) [inline tests/06Folders.spec.js:22]: I think you don't need this line

File: tests/06Folders.spec.js [hunk 2]
```diff
@@ -1,25 +1,24 @@
...
+        await documentsPage.locators.getToaster().waitFor({ state: 'visible' });
+        await documentsPage.locators.getToaster().waitFor({ state: 'hidden' });
 
-        await expect(templatesActivePage.locators.getToaster()).toHaveText('Folder deleted successfully.');
+        await documentsPage.clickOptionsBtn();
+        await documentsPage.clickDeleteBtn();
+        await documentsPage.clickYesDeleteBtn();
+        await documentsPage.locators.getToaster().waitFor({ state: 'visible' });
+
+        await expect(documentsPage.locators.getToaster()).toHaveText('Folder deleted successfully.');
```
Comment (S-Lanchik) [inline tests/06Folders.spec.js:20]: I would add assertion to check that the folder does not exist in the document table.  It makes sure that folder is deleted for sure.

Comment (S-Lanchik) [reply tests/06Folders.spec.js:20]: But maybe this assertion is enough

**Comment by S-Lanchik** [comment]
Everything else looks good to me.

**Review by S-Lanchik — APPROVED** [review]

File: tests/06Folders.spec.js (hunk 1 above)
Comment (verafes) [reply tests/06Folders.spec.js:22]: Until further notice, our current requirement is to navigate back to the home (/sign) page after removing all data.

**Review by SvetlanaKanel — APPROVED** [review]
lgtm

**Pull request merged**